src/Storage/TSStorageManager.h

```diff
@@ -21,9 +17,15 @@ extern NSString *const TSUIDatabaseConnectionDidUpdateNotification;
 @interface TSStorageManager : NSObject
 
 + (instancetype)sharedManager;
+
+/**
+ * Database password might not be accessible if the user just restarted the device and hasn't unlocked. If that's the
+ * case, we can't usefully start the app until they unlock.
```

Comment:
The db password is created even before user registers, right? What if user de-registers? Is the db ever going to be deleted in that case?